CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog for Pulse3D
 =====================
 
+0.33.12 (unreleased)
+--------------------
+
+Fixed:
+^^^^^^
+- Fixed peak finding error for wells with no peaks detected
```

Comment:
This isn't exactly what's happening, the issue is when no peaks are found at least one segment size away from the end of the data